db_upgrade_scripts/mosip_regprc/dml/regprc-transaction_type_rollback.csv

```diff
@@ -0,0 +1,25 @@
+code,descr,lang_code,is_active,cr_by,cr_dtimes
+CREATE,transaction_done,eng,TRUE,MOSIP_SYSTEM,now()
+DELETE,delete transaction,eng,TRUE,MOSIP_SYSTEM,now()
+FAILED,transaction failed,eng,TRUE,MOSIP_SYSTEM,now()
+UPDATE,transaction_done,eng,TRUE,MOSIP_SYSTEM,now()
+DEMO,transaction_done,eng,TRUE,MOSIP_SYSTEM,now()
+BIO,transaction_done,eng,TRUE,MOSIP_SYSTEM,now()
+PACKET_RECEIVER,transaction done,eng,TRUE,MOSIP_SYSTEM,now()
+UPLOAD_PACKET,transaction done,eng,TRUE,MOSIP_SYSTEM,now()
+VALIDATE_PACKET,transaction done,eng,TRUE,MOSIP_SYSTEM,now()
+VIRUS_SCAN,transaction done,eng,TRUE,MOSIP_SYSTEM,now()
+OSI_VALIDATE,transaction done,eng,TRUE,MOSIP_SYSTEM,now()
+EXTERNAL_INTEGRATION,transaction done,eng,TRUE,MOSIP_SYSTEM,now()
+DEMOGRAPHIC_VERIFICATION,transaction done,eng,TRUE,MOSIP_SYSTEM,now()
+MANUAL_VERIFICATION,transaction done,eng,TRUE,MOSIP_SYSTEM,now()
+BIOGRAPHIC_VERIFICATION,transaction done,eng,TRUE,MOSIP_SYSTEM,now()
+UIN_GENERATOR,transaction done,eng,TRUE,MOSIP_SYSTEM,now()
+NOTIFICATION,transaction done,eng,TRUE,MOSIP_SYSTEM,now()
+PRINT_SERVICE,transaction done,eng,TRUE,MOSIP_SYSTEM,now()
+PRINT_POSTAL_SERVICE,transaction done,eng,TRUE,MOSIP_SYSTEM,now()
+QUALITY_CHECK,transaction done,eng,TRUE,MOSIP_SYSTEM,now()
+BIOMETRIC_AUTHENTICATION,transaction done,eng,TRUE,MOSIP_SYSTEM,now()
+PACKET_REPROCESS,transcation done,eng,TRUE,MOSIP_SYSTEM,now()
+SECUREZONE_NOTIFICATION,transaction notification,eng,TRUE,MOSIP_SYSTEM,now()
+PACKET_CLASSIFICATION,transaction done,eng,TRUE,MOSIP_SYSTEM,now()
```

Comment:
As discussed, lets create separate insert statements in the upgrade and delete statatements in the rollback scripts to take care of this. Lets also avoid removing the constraints and readding them because of these tructions to this table.

Reply:
fixed